arrow-libs/core/arrow-continuations/build.gradle

```
@@ -12,6 +12,11 @@ apply plugin: 'kotlinx-atomicfu'
 
 kotlin {
     sourceSets {
+        commonMain {
+            dependencies {
+                compileOnly "org.jetbrains.kotlin:kotlin-stdlib-common:$KOTLIN_VERSION"
```

```suggestion
                compileOnly "org.jetbrains.kotlin:kotlin-stdlib:$KOTLIN_VERSION"
```

It will be solved as `...-common`.

---

The same proposal for other cases :raised_hands:

---

Included in #2404. Resolved conversation!